The_Seven_Kingdoms/localisation/001_religion_descs.csv

```diff
@@ -5,10 +5,10 @@ the_mother_DESC;The Mother is a heavenly aspect of the Seven, whom represents me
-vulture_god_DESC;The Vulture is the Henotheistic faith of the Stone Dornishmen of the Red Mountains. Little is known of this faith, as it was quickly extiquished during the Andal and Rhoyane invasions of Dorne.;;;;;;;;;;;;;x
+vulture_god_DESC;The Vulture is the Henotheistic faith of the Stone Dornishmen of the Red Mountains. Little is known of this faith, as it was quickly extinguished during the Andal and Rhoyane invasions of Dorne.;;;;;;;;;;;;;x
 old_gods_DESC;The Old Gods of the Forest, or simply the Old Gods, is the traditional animistic faith of the Children of the Forest, and then the First Men. It is vastly different from the Seven, in that there is no rite, no uniform prayer, and no organized clergy. What is uniform of the religion however, is the sacredness of the godswood, and the weirwood trees that normally inhabit them. These places are usually where the followers of the Old Gods take to pray, and to listen to their gods. The Gods of the faith are nameless and numberless, and the children believed that each weirwood was inhabited by one of the gods.;;;;;;;;;;;;;x
 old_gods_beyond_DESC;Much like their southern cousins, the First Men that inhabit the lands beyond the wall worship the Old Gods. Unlike their southern cousins however, they are more primal and rural, with the gods being more natural beings that inhabit the lands around them, such as springs and the sacred weirwoods.;;;;;;;;;;;;;x
-old_gods_beyond_white_DESC;In the farthest reaches beyond the wall live savage, almost animalistic men, whom live a harsh, cold life near the land of Always Winter. Here, the influence of the Others is most felt, and as such, the men nearest their domains offer sacrifices and worship to them, so that they might be spared their icy wrath. Besides legend, little conclusive information is known about the Others, but it is generally accepted that a deity called the Great Other is their peer, and thus is the one whom recieves the sacrifices and and prayers.;;;;;;;;;;;;;x
+old_gods_beyond_white_DESC;In the farthest reaches beyond the wall live savage, almost animalistic men, whom live a harsh, cold life near the land of Always Winter. Here, the influence of the Others is most felt, and as such, the men nearest their domains offer sacrifices and worship to them, so that they might be spared their icy wrath. Besides legend, little conclusive information is known about the Others, but it is generally accepted that a deity called the Great Other is their peer, and thus is the one whom receives the sacrifices and prayers.;;;;;;;;;;;;;x
```

Comment:
Changed "recieves the sacrifices and and prayers." to "**receives** the sacrifices **and** prayers." If it's not clear.